packages/@lwc/template-compiler/src/__tests__/parser.spec.ts

```diff
@@ -33,18 +33,18 @@ function parseTemplate(src: string): any {
 describe('parsing', () => {
```

Comment:
I would like to get rid of this test file and migrate it to a fixture-based approach now that the template compiler produces a serializable AST. On top of producing [`expected.js` and `metadata.json`](https://github.com/salesforce/lwc/blob/master/packages/%40lwc/template-compiler/src/__tests__/fixtures.spec.ts#L32-L33), we should also produce an `ast.json` file to make sure the AST remains stable over time. 

As this change would certainly introduce a lot of changes and make this PR harder to review, I think it is better to do this in a follow-up PR.